css-view-transitions-1/Overview.bs

```diff
@@ -1091,6 +1091,87 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
+			and return.
+
+		1. Set |document|'s [=document/transition suppressing rendering=] to true.
+
+		1. [=Queue a global task=] on the [=DOM manipulation task source=],
+			given |transition|'s [=relevant global object=],
+			to execute the following steps:
+
+				Note: A task is queued here because the texture read back in [=capturing the image=] may be async,
+					although the render steps in the HTML spec act as if it's synchronous.
+
+			1. If |transition|'s [=ViewTransition/phase=] is "`done`", then abort these steps.
+
+				Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
+					The [=skip the view transition=] steps [=call the update callback=],
+					ensuring the |transition|'s [=ViewTransition/update callback=] is always called.
+
+			1. [=Call the update callback=] of |transition|.
+
+			1. [=promise/React=] to |transition|'s [=ViewTransition/update callback done promise=]:
+
+				* If the promise does not settle within an implementation-defined timeout, then:
+
+					1. If |transition|'s [=ViewTransition/phase=] is "`done`", then return.
+
+						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
+
+					1. [=Skip the view transition=] |transition| with a "{{TimeoutError}}" {{DOMException}}.
+
+				* If the promise was rejected with reason |reason|, then:
+
+					1. If |transition|'s [=ViewTransition/phase=] is "`done`", then return.
+
+						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
+
+					1. [=Skip the view transition=] |transition| with |reason|.
+
+				* If the promise was resolved, then:
+
+					1. If |transition|'s [=ViewTransition/phase=] is "`done`", then return.
+
+						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
+
+					1. If |transition|'s [=ViewTransition/initial snapshot root size=] is not equal to the [=snapshot root size=],
+						then [=skip the view transition=] for |transition|, and return.
+
+					1. Set [=document/transition suppressing rendering=] to false.
+
+					1. [=Capture the new state=] for |transition|.
+
+						If failure is returned, then [=skip the view transition=] for |transition| with an "{{InvalidStateError}}" {{DOMException}} in |transition|'s [=relevant Realm=],
+						and return.
+
+					1. [=Setup transition pseudo-elements=] for |transition|.
+
+					1. [=Update pseudo-element styles=] for |transition|.
+
+						Note: The above steps will require running document lifecycle phases,
+							to compute information calculated during style/layout.
+
+					1. Set |transition|'s [=ViewTransition/phase=] to "`animating`".
+
+					1. [=Resolve=] |transition|'s [=ViewTransition/ready promise=].
+	</div>
+
+### [=Capture the old state=] ### {#capture-old-state-algorithm}
```

Comment:
I've groups some algorithms with a parent algorithm, if they're only ever called by the parent. This gives us a more hierarchical heading structure.